lib/std/debug.zig

```diff
@@ -264,6 +264,8 @@ pub fn panicExtra(trace: ?*const builtin.StackTrace, first_trace_addr: ?usize, c
                 defer held.release();
 
                 const stderr = io.getStdErr().writer();
+                const current_thread_id = std.Thread.getCurrentThreadId();
```

Comment:
Should probably check `if (!builtin.single_threaded)`

Reply:
What should it print then? Nothing? Something like `main thread panicked: `?